Makefile

```diff
@@ -0,0 +1,6 @@
+# Setup
+setup: install
+
+# Install dependencies
+install:
+	brew install xcbeautify
```

Comment:
Why don't we use Cocoapod to install `xcbeautify`
```rb
pod 'xcbeautify'
```

Reply:
My bad 🙈 , it can be simply installed with the pod. Thank you so much 🙇. I updated it at https://github.com/nimblehq/ios-templates/pull/452/commits/0e9dfa4cf4120884fec301b579539208fac97e31

Reply:
@suho @vnntsu @blyscuit  I just recalled why I came up with installing `xcbeautify` by brew instead of `Cocoapod` at the first commit🙏 

It may be some error with xcodeFormtter in fastlane on CI when installing xcbeautify by Cocoapod, [a thread](https://github.com/fastlane/fastlane/issues/19924) raised this once.

I also experimented with some adjusting below. The local run is success, but all failure on CI with the same error not found xcbeautify.

Test using pod + xcodebuildFormatter: xcbeautify
**Job:** https://github.com/nimblehq/ios-templates/actions/runs/4382401570/jobs/7671406433
**Result:** Failure with `error: sh: xcbeautify: command not found`.

Test using pod without setting xcodebuildFormatter, the default value is xcbeautify
**Job:** https://github.com/nimblehq/ios-templates/actions/runs/4382568797/jobs/7671765515
**Result:** Failure with `error: sh: xcbeautify: command not found`.

Therefore, I came up with the solution xcbeautify installed by brew on CI. If you have any suggestions or concerns, feel free to discuss more. Thank you a lot 🙇 .

Reply:
Have you try pinpointing the location of the xcbeautify binary when installed with pods? https://docs.fastlane.tools/best-practices/xcodebuild-formatters/
```
  xcodebuild_formatter: "/pods/xcbeautify/xcbeautify" // not sure what is the binary path
```

Reply:
@suho @blyscuit @vnntsu It works like a charm 👯